docs/root/intro/arch_overview/http/websocket.rst

```diff
@@ -39,16 +39,18 @@ downstream_cx_upgrades_total and downstream_cx_upgrades_active
 Handling H2 hops
 ^^^^^^^^^^^^^^^^
 
-Envoy supports tunneling WebSockets over H2 streams for deployments that prefer a uniform
-H2 mesh throughout; this enables, for example, a deployment of the form:
+While H2 support for WebSockets is off by default, Envoy does supports tunneling WebSockets over
```

Comment:
Nit: prefer HTTP/2 over H2 in docs (I know this was there before, if you could search and replace it would be great)
Nit: s/does supports/does support/